src/main/webapp/script.js

```diff
@@ -147,9 +147,6 @@ function deleteGrocery(grocery) {
   fetch('/delete-grocery', {method: 'POST', body: params});
 }
 
-function isEditableGrocery(grocery) {
-  return grocery.userMatch || !grocery.email; 
-
```

Comment:
is there another isEditableGrocery function somewhere else? This function is needed for the delete button.

Reply:
Oh I see now nevermind

Reply:
yeah there was a duplicate somehow so I just removed one